spec/mailers/purchase_or_donate_mailer_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "spec_helper"
+
+  RSpec.describe PurchaseOrDonateMailer, type: :mailer do
+    class PurchaseOrDonateMailerTest < ActionMailer::TestCase
+    test "donate" do
+      # Send the email, then test that it got queued
+      email = PurchaseOrDonateMailer.please_give_email('me@example.com').deliver_now
+      expect(ActionMailer::Base.deliveries)not_to be_empty
```

Comment:
there should be a period after the parens here i.e. `expect(ActionMailer::Base.deliveries).not_to be_empty`